packages/@webex/plugin-cc/README.md

```
@@ -0,0 +1,71 @@
+  yarn add @webex/plugin-cc
+```
+
+```javascript
+  import ContactCenter from '@webex/plugin-cc'
+```
+# CDN
+```javascript
+  <script src="../contact-center.min.js"></script>
+```
+
+### Kitchen Sink App
```

Markdownlint (0.37.0)

Expected: h2; Actual: h3
Heading levels should only increment by one level at a time

(MD001, heading-increment)